docs(chart): add kb for custom stacked column labels (#1712)
telerik/blazor-docs: knowledge-base/chart-stacked-series-sum-label.md

        <ChartSeries Data="@SailorsData"
                     Name="Sailors Features"
                     Field="@nameof(ChartModel.Features)"
                     CategoryField="@nameof(ChartModel.DateValue)">
            <ChartSeriesStack Group="Sailors" Enabled="true" />
            <ChartSeriesLabels Visible="true"
                               Template="stackSumLabel">
            </ChartSeriesLabels>
            <ChartSeriesTooltip Visible="true">
                <Template>
                    @context.FormattedValue features
                </Template>
            </ChartSeriesTooltip>
        </ChartSeries>
    </ChartSeriesItems>

    <ChartCategoryAxes>
        <ChartCategoryAxis Type="@ChartCategoryAxisType.Date"
                           BaseUnit="@ChartCategoryAxisBaseUnit.Months">
            <ChartCategoryAxisLabels>
                <ChartCategoryAxisLabelsMargin Top="80" />
            </ChartCategoryAxisLabels>
        </ChartCategoryAxis>
    </ChartCategoryAxes>

    <ChartValueAxes>
        <ChartValueAxis Max="20" MajorUnit="4" />
    </ChartValueAxes>

    <ChartLegend Visible="true" Position="@ChartLegendPosition.Top" />

</TelerikChart>

> @* Move JavaScript to a JS file in production apps *@

Author: ```suggestion
@* Move JavaScript to a JS file in production apps *@
```

> @* Move JavaScript to separate JS file in production *@

Author: ```suggestion
@* Move JavaScript to a JS file in production apps *@
```

            <ChartSeriesTooltip Visible="true" />
        </ChartSeries>
        <ChartSeries Data="@CowboysData"
                     Name="Features"
                     Field="@nameof(ChartModel.Features)"
                     CategoryField="@nameof(ChartModel.DateValue)">
            <ChartSeriesLabels Visible="true"
                               Position="@ChartSeriesLabelsPosition.InsideBase"
                               Rotation="-90"
                               Template="featuresLabel"
                               Background="transparent">
                <ChartSeriesLabelsMargin Top="75" />
            </ChartSeriesLabels>
            <ChartSeriesTooltip Visible="true" />
        </ChartSeries>
    </ChartSeriesItems>

    <ChartCategoryAxes>
        <ChartCategoryAxis Type="@ChartCategoryAxisType.Date"
                           BaseUnit="@ChartCategoryAxisBaseUnit.Months">
            <ChartCategoryAxisLabels>
                <ChartCategoryAxisLabelsMargin Top="70" />
            </ChartCategoryAxisLabels>
        </ChartCategoryAxis>
    </ChartCategoryAxes>

    <ChartLegend Visible="true" Position="@ChartLegendPosition.Top" />

</TelerikChart>

> @* Move JavaScript to separate JS file in production *@

Author: ```suggestion
@* Move JavaScript to a JS file in production apps *@
```

<script suppress-error="BL9992">
    function bugsLabel(context) {
        return "Bugs";
    }
    function featuresLabel(context) {
        return "Features";
    }
</script>

@code {
    private List<ChartModel> CowboysData { get; set; } = new List<ChartModel>();

    protected override void OnInitialized()
    {
        var rnd = new Random();

        for (int i = 0; i < 6; i++)
        {
            CowboysData.Add(new ChartModel()
            {
                DateValue = DateTime.Today.AddMonths(-i),
                Bugs = rnd.Next(5, 10),
                Features = rnd.Next(5, 10)
            });
        }

        base.OnInitialized();
    }

    public class ChartModel
    {
        public DateTime DateValue { get; set; }
        public int Bugs { get; set; }
        public int Features { get; set; }
    }
}
````

## Notes

> Chart label templates require a different configuration for product versions before 4.5.0. If you are using an older version, then [download the PDF documentation](https://www.telerik.com/account/downloads/product-download?product=BLAZOR) for your version, or [browse an older version of the Chart Label Templates documentation](https://github.com/telerik/blazor-docs/blob/4.4.0/components/chart/labels-template-and-format.md#templates).

Reviewer: ```suggestion
Chart label templates require a different configuration for [Telerik UI for Blazor versions](https://www.telerik.com/support/whats-new/blazor-ui/release-history) before 4.5.0. If you are using an earlier version, then [download the PDF documentation](https://www.telerik.com/account/downloads/product-download?product=BLAZOR) for your version, or [browse an older version of the Chart Label Templates documentation](https://github.com/telerik/blazor-docs/blob/4.4.0/components/chart/labels-template-and-format.md#templates).
```

> Chart label templates require different configuration for product versions before 4.5.0. If you are using an older version, then [download the PDF documentation](https://www.telerik.com/account/downloads/product-download?product=BLAZOR) for your version, or [browse an older version of the Chart Label Templates documentation](https://github.com/telerik/blazor-docs/blob/4.4.0/components/chart/labels-template-and-format.md#templates).

Author: ```suggestion
Chart label templates require a different configuration for product versions before 4.5.0. If you are using an older version, then [download the PDF documentation](https://www.telerik.com/account/downloads/product-download?product=BLAZOR) for your version, or [browse an older version of the Chart Label Templates documentation](https://github.com/telerik/blazor-docs/blob/4.4.0/components/chart/labels-template-and-format.md#templates).
```
